submissions/best-of-its-genre-part-2-nanogamer7/influence.md

```diff
@@ -0,0 +1,62 @@
+
+{screenshot(s) cause images are good to break up text}
+
+### Part 1: The Big Three
+
+When you ask someone to name some automation games other than Factorio, the most common response you'll get are Satisfactory and Dyson Sphere Program. Together with Factorio, they are even regarded as the (un-)holy trinity of automation by some players.[^1] But what makes these games stand out from the rest (apart from their rating, the highest among all games on steam with the 'automation' tag[^2])?
+
+{screenshot of ratings?}
+
+- All three games are set in an open world, and you, the player, are an entity in it building the factory. Neither of them is required for automation, as for example [shapez.io](https://store.steampowered.com/app/1318690/shapezio/) perfectly works without a playable character, and there a plenty of games like [Opus Magnum](https://store.steampowered.com/app/558990/Opus_Magnum/) that are in a level-based puzzle environment.
+- Your progression is mainly through a skill/technology tree (even if it isn't actually tree shaped in Satisfactory)—special technology to send through a space elevator in Satisfactory, research matrixes in DSP, and finally, the bottle-shaped science packs in Factorio. Most open world (survival) games feature some sort of skill tree, but acquiring new points rarely features a connection to the game world, while your world, your factory is to some degree completely build around the tech tree; your progression in the skill tree isn't based on your progress in the rest of the game, it **is** your progress in the game.
+
+{gallery, the 3 different research trees, +some "science" setups}
+
+- And finally, all three games games mainly challenge your knowledge and understanding of logistics. While the spaghetti of some bases might suggest otherwise, space isn't actually an issue (or at least not a big one) due to the open world, but getting massive amounts of raw resources from a mining outpost back to your main base, and maybe even moving production to said outpost to export the most dense/compact products (see [Alt-F4 #7](https://alt-f4.blog/ALTF4-7/#megabase-thinking-lonewolf)). This challenge might be a direct result of the open world nature of the games, but especially long range trucks and trains provide a much needed "unknow" variable to the defined ratios—unknown as in not as obvious as items per second on belts—as well as a challenge, a bottle neck scaling inversely with production.
+
+Apart from the similarities in gameplay, the three games also feature a remarkably similar scenario of being dropped on an empty planet with only some basic resources to start with; although I don't think there are (m)any other themes that would explain why you need to make **everything** from scratch, at least none that wouldn't sound artificial.
+
```

Comment:
I think you should just pull together your points on this chapter before you go to the next one. Something like:

> These are some big aspects that ties these three games together. There might be many more ties, but if you try to take it down to basics then this is the main simularities. This means automation can be defined by these points. But does it hold up to other games that we do not see as automation?